projects/Svelte.py

```diff
@@ -0,0 +1,42 @@
+from interfaces.ProjectInterface import ProjectInterface
+from interfaces.TestError import TestError
+from interfaces.LintError import LintError
+from helpers.ProjectHelper import (
+    install_npm_packages, fix_eslint_issues, 
+    get_eslint_errors, get_vitest_errors
+)
+import subprocess
+
+
+class Svelte(ProjectInterface):
+    @property
+    def path(self):
+        return 'repos/svelte'
+
+    @property
+    def code_dir(self):
+        return '/packages'
```

Comment:
Actually the project source code is in the src folder. It is not a big problem since, as of now, the other child directories of `/packages` do not contain any code, thus lizard won't eventually find "false positives". But maybe this would be more clarifying for the future as well.

```suggestion
        return '/packages/svelte/src'
```